lib/Controller/EndpointController.php

```diff
@@ -62,17 +62,25 @@ public function __construct(string $appName,
 
 
 	/**
+	 * @param string $rootPassword
+	 *
 	 * @return DataResponse
```

Comment:
`@PasswordConfirmationRequired` is missing, otherwise you can directly call the API without the password